packages/auth/src/AuthService.ts

```diff
@@ -17,40 +18,48 @@ export class AuthService {
    * @param initOptions Initialization options.
    * @returns A promise to set functions to be invoked on success or error.
    */
-  public init(initOptions: KeycloakInitOptions): KeycloakPromise<boolean, KeycloakError> {
+  public init(initOptions: KeycloakInitOptions): Bluebird<boolean> {
     if (!initOptions.onLoad) {
       initOptions.onLoad = "check-sso";
     }
-    return this.auth.init(initOptions);
+    return new Bluebird((resolve, reject) => {
+      return this.auth.init(initOptions).error(reject).success(resolve);
+    });
   }
 
   /**
    * Loads the user's profile.
    * @returns A promise to set functions to be invoked on success or error.
    */
-  public loadUserProfile(): KeycloakPromise<KeycloakProfile, void> {
-    return this.auth.loadUserProfile();
+  public loadUserProfile(): Bluebird<KeycloakProfile> {
+    return new Bluebird((resolve, reject) => {
+      return this.auth.loadUserProfile().error(reject).success(resolve);
+    });
   }
 
   /**
    * Redirects to login form.
    * @param options Login options.
    */
-  public login(): KeycloakPromise<void, void> {
-    return this.auth.login();
+  public login(): Bluebird<void> {
+    return new Bluebird((resolve, reject) => {
+      return this.auth.login().error(reject).success(resolve);
+    });
   }
 
   /**
    * Redirects to logout.
    * @param options Logout options.
    * @param options.redirectUri Specifies the uri to redirect to after logout.
    */
-  public logout(): KeycloakPromise<void, void> {
-    return this.auth.logout();
+  public logout(): Bluebird<void> {
+    return new Bluebird((resolve, reject) => {
+      return this.auth.logout().error(reject).success(resolve);
+    });
   }
 
-  public isAuthenticated(): boolean | undefined {
-    return this.auth.authenticated;
+  public isAuthenticated(): boolean  {
+    return !!this.auth.authenticated;
```

Comment:
Why the double negation?

Reply:
Addresses previous issue. Unwraps undefined.

Reply:
Could we do `this.auth.authenticated as boolean` instead?

Reply:
Or even `return value ? true : false;` if the type must be really boolean

Reply:
@josemigallas This comes wrong from keycloak. `return value ? true : false;` has the same effect. Let's not go into holly war with the style on PR.